### ISSUE TITLE
RADEON VII refuse to start with latest drivers

### ISSUE THREAD

**Issue by slappy6212** [andru-kun/wildrig-multi#278]
My Radeon VII refuse to mine and give the following message :
Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for kawpow...or any other algo's
tried version 0.39.9c and 0.39.9b.

**Comment by andru-kun** [comment]
OS? Drivers versiin?

**Comment by slappy6212** [comment]
Windows 11 drivers 23.11.1 but i tried the previous drivers versions with same  errors.Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Andrii ***@***.***> Date : 2023-12-12  2 h 18 a.m.  (GMT-05:00) À : andru-kun/wildrig-multi ***@***.***> Cc : slappy6212 ***@***.***>, Author ***@***.***> Objet : Re: [andru-kun/wildrig-multi] RADEON VII refuse to start with latest drivers (Issue #278) 
OS? Drivers versiin?

**Comment by andru-kun** [comment]
can you send logs when miner starts? I want to see what architecture it shows there. Also does it work with drivers from 2019 or 2020?

**Comment by slappy6212** [comment]
Yes it work up to sept 2020...but i also need new amd drivers for other software on my pc this is why i can no longer downgrade amd drivers. I am at work now. Need more time for the log.Thanks for your helpSteveEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Andrii ***@***.***> Date : 2023-12-12  7 h 37 a.m.  (GMT-05:00) À : andru-kun/wildrig-multi ***@***.***> Cc : slappy6212 ***@***.***>, Author ***@***.***> Objet : Re: [andru-kun/wildrig-multi] RADEON VII refuse to start with latest drivers (Issue #278) 
can you send logs when miner starts? I want to see what architecture it shows there. Also does it work with drivers from 2019 or 2020?

**Comment by slappy6212** [comment]
Is it the log you need ?

**Comment by slappy6212** [comment]
Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Andrii ***@***.***> Date : 2023-12-12  7 h 37 a.m.  (GMT-05:00) À : andru-kun/wildrig-multi ***@***.***> Cc : slappy6212 ***@***.***>, Author ***@***.***> Objet : Re: [andru-kun/wildrig-multi] RADEON VII refuse to start with latest drivers (Issue #278) 
can you send logs when miner starts? I want to see what architecture it shows there. Also does it work with drivers from 2019 or 2020?

**Comment by slappy6212** [comment]
[FILE.txt](https://github.com/andru-kun/wildrig-multi/files/13649708/FILE.txt)

**Comment by andru-kun** [comment]
> [FILE.txt](https://github.com/andru-kun/wildrig-multi/files/13649708/FILE.txt)

Ok, thanks, I will take a look on this

**Comment by andru-kun** [comment]
try version 0.40.0

**Comment by slappy6212** [comment]
Not working.Envoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Andrii ***@***.***> Date : 2023-12-15  6 h 39 a.m.  (GMT-05:00) À : andru-kun/wildrig-multi ***@***.***> Cc : slappy6212 ***@***.***>, Author ***@***.***> Objet : Re: [andru-kun/wildrig-multi] RADEON VII refuse to start with latest drivers (Issue #278) 
try version 0.40.0

**Comment by andru-kun** [comment]
Well, then probably nothing I can do here. They changed kernel format for Windows, and I won't compile kernels specifically for 1 gpu, sorry.

**Comment by slappy6212** [comment]
Here is what i have.[07:51:05] Initializing GPUs...[07:51:05] GPU #0: AMD Radeon VII [busID: 3]  [arch: gfx906][07:51:05] threads: 1, intensity: 1920, cu: 60, memory: 16368Mb[07:51:06] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for blake[07:51:07] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for blake2b[07:51:08] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for bmw[07:51:09] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for cn_dark[07:51:10] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for cn_fast[07:51:11] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for cn_lite[07:51:12] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for cn_turtle[07:51:13] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for cn_common[07:51:14] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for cubehash[07:51:15] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for echo[07:51:16] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for fugue[07:51:17] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for gost[07:51:19] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for groestl[07:51:20] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for hamsi[07:51:21] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for haval[07:51:22] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for jh[07:51:23] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for keccak[07:51:24] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for lane[07:51:25] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for luffa[07:51:26] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for sha256[07:51:27] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for sha512[07:51:28] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for shabal[07:51:29] Error CL_BUILD_PROGRAM_FAILURE when calling clBuildProgram for shavite[07:51:30] Error CL_BUILD_PROGRAM_FAILURE whenEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Andrii ***@***.***> Date : 2023-12-15  7 h 40 a.m.  (GMT-05:00) À : andru-kun/wildrig-multi ***@***.***> Cc : slappy6212 ***@***.***>, Author ***@***.***> Objet : Re: [andru-kun/wildrig-multi] RADEON VII refuse to start with latest drivers (Issue #278) 
Well, then probably nothing I can do here. They changed kernel format for Windows, and I won't compile kernels specifically for 1 gpu, sorry.
